src/proteus/proteus.py

```diff
@@ -423,27 +420,24 @@ def start(self, *, resume: bool = False):
                 loop_counter["total"] == 0
             ):
                 sspec_prev = hf_row["Time"]
-
-                # Remove old spectral file if it exists
-                safe_rm(spfile_path)
-                safe_rm(spfile_path + "_k")
+                update_stellar_spectrum = True
 
                 log.info("Updating stellar spectrum")
                 match self.config["star_model"]:
                     case 0:
                         synthetic = mors.synthesis.CalcScaledSpectrumFromProps(
                             star_struct_modern, star_props_modern, hf_row["age_star"] / 1e6
                         )
-                        fl = synthetic.fl  # at 1 AU
-                        wl = synthetic.wl
+                        hf_row["fl"] = synthetic.fl  # at 1 AU
```

Comment:
I am not sure that this is a good idea. At the moment `hf_row` is only designed to store scalar quantities, since it is then appended to the end of the helpfile dataframe and CSV file. These `fl` and `wl` arrays are 1D, so it will not be possible to simply append them to the helpfile in this manner.

I would suggest keeping these variables standalone for now, and we can later think about handling them better.

Reply:
Wanted to lighten the call to the functions but I get your point. Fixed in new commit.